pkgs/tools/backup/pyznap/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonApplication
+, pythonOlder
+, fetchPypi
+, paramiko
+, configparser
+}:
+
+buildPythonApplication rec {
+  pname = "pyznap";
+  version = "1.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "9ac0da5d7f6461d1d6f128362786e297144b415f9e3a2f1835642ab3dda82d55";
+  };
+
+  propagatedBuildInputs = [ configparser paramiko ];
+
+  doCheck = false;
```

Comment:
Does it have no checks or are they disable for a different reason? Please add a comment to make that clear.

Reply:
In that case just add the extra packages needed for testing to ```checkInputs```.

Reply:
Further digging reveals that the PyPI release of the package doesn't include the tests.